components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/ProfileOperationDAOImpl.java

```diff
@@ -37,22 +37,46 @@
 
     public int addOperation(Operation operation) throws OperationManagementDAOException {
         PreparedStatement stmt = null;
+        ByteArrayOutputStream bao = null;
+        ObjectOutputStream oos = null;
+
         int operationId;
         try {
             operationId = super.addOperation(operation);
             operation.setCreatedTimeStamp(new Timestamp(new java.util.Date().getTime()).toString());
             operation.setId(operationId);
             operation.setEnabled(true);
-            ProfileOperation profileOp = (ProfileOperation) operation;
+            //ProfileOperation profileOp = (ProfileOperation) operation;
```

Comment:
Remove commented line. And, can we change this to Java try with resource?

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275